Release id scoped by the ocid AND the version (#1242)
open-contracting/standard: docs/history/changelog.md

  * `Organization.id`: "This field need only be unique within the scope of the contracting process, but **should** be built with the following structure {identifier.scheme}-{identifier.id}(-{department-identifier}) where an organization identifier is available" ("may" replaced with "should")
  * `Period.durationInDays`:  "If a startDate and endDate are set, this field, if used, **must** be equal to the difference between startDate and endDate. Otherwise, if a startDate and maxExtentDate are set, this field, if used, **must** be equal to the difference between startDate and maxExtentDate." ("should" replaced with "must")
  * `Contract.items`: "If the items contracted are identical to the items awarded, this field **should** be omitted." (rephrased)

> * [#1125](https://github.com/open-contracting/standard/pull/1125) Add `weight` to `Item.unit`
> * [#1115](https://github.com/open-contracting/standard/pull/1115) Clarify when to have multiple suppliers per award
> * [#1067](https://github.com/open-contracting/standard/pull/1067) Clarify the description of `Publisher.name` to indicate that it is the organization or department responsible for publishing the OCDS version of the data.

Reviewer: Appears elsewhere in changelog.

```suggestion
```

> * [#1242](https://github.com/open-contracting/standard/pull/1242) Expand the scope of release `id` to ocid and OCDS version (was ocid only), allowing the publication of datasets for different versions of OCDS.

Reviewer: ```suggestion
* [#1242](https://github.com/open-contracting/standard/pull/1242) Narrow the uniqueness scope of a release's `id` to its `ocid` and OCDS version (was `ocid` only), to allow the publication of the same release for different versions of OCDS.
```

> * [#1242](https://github.com/open-contracting/standard/pull/1242) Add OCDS version on top of ocid as a merging facet.

Reviewer: ```suggestion
* [#1242](https://github.com/open-contracting/standard/pull/1242) Merging: Clarify that the releases to merge must use the same version of OCDS.
```
